msbuild/Xamarin.MacDev.Tasks/Tasks/ResolveNativeReferencesBase.cs

```diff
@@ -497,5 +506,28 @@ internal static bool TryResolveXCFramework (TaskLoggingHelper log, PDictionary p
 			log.LogError (MSBStrings.E0175 /* No matching framework found inside '{0}'. SupportedPlatform: '{0}', SupportedPlatformVariant: '{1}', SupportedArchitectures: '{2}'. */, xcframeworkPath, platformName, variant, architectures);
 			return false;
 		}
+
+		public void Cancel ()
+		{
+			if (ShouldExecuteRemotely ())
+				BuildConnection.CancelAsync (BuildEngine4).Wait ();
+		}
+
+		public bool ShouldCopyToBuildServer (ITaskItem item) => true;
+
+		public bool ShouldCreateOutputFile (ITaskItem item)
+		{
+			if (NativeFrameworks is not null && Array.IndexOf (NativeFrameworks, item) >= 0) {
+				// Don't copy any resolved frameworks back to Windows, because
+				// 1. They're not used in Inputs/Outputs, so the lack of them won't affect anything
+				// 2. They may be directories, and as such we'd have to expand them to (potentially numerous and large) files to copy them (uselessly) to Windows.
+				// 3. They may contain symlinks, which may not work correctly on Windows.
+				return false;
```

Comment:
I see there are two output properties in this class. Returning false means none of those output properties will be considered for creating the files on Windows.
I’m just trying to understand why we would only want to create output files when there are no frameworks. Is it okay to discard the TouchedFiles output when there are frameworks?
On the other hand, our outputs are always empty files, we never copy a real output file.

Reply:
I think I can change this to never create an output file, that would actually make more sense.